R/uniDimFrequentistCalcCoeff.R

```diff
@@ -569,43 +569,43 @@
   return(out)
 }
 
-.frequentistItemMean <- function(jaspResults, dataset, options, model) {
-  if (!is.null(.getStateContainerF(jaspResults)[["itemMeanObj"]]$object))
-    return(.getStateContainerF(jaspResults)[["itemMeanObj"]]$object)
+.frequentistMeanItem <- function(jaspResults, dataset, options, model) {
+  if (!is.null(.getStateContainerF(jaspResults)[["meanItemObj"]]$object))
+    return(.getStateContainerF(jaspResults)[["meanItemObj"]]$object)
 
-  out <- model[["itemMean"]]
+  out <- model[["meanItem"]]
   if (is.null(out))
     out <- list()
   # is box even checked?
-  if (options[["itemMean"]]  && is.null(model[["empty"]])) {
+  if (options[["meanItem"]]  && is.null(model[["empty"]])) {
     out[["itemDropped"]] <- colMeans(dataset, na.rm = TRUE)
 
     if (options[["disableSampleSave"]])
       return(out)
 
     stateContainer <- .getStateContainerF(jaspResults)
-    stateContainer[["itemMeanObj"]] <- createJaspState(out, dependencies = c("itemMean"))
+    stateContainer[["meanItemObj"]] <- createJaspState(out, dependencies = c("meanItem"))
   }
   return(out)
 }
 
-.frequentistItemSd <- function(jaspResults, dataset, options, model) {
-  if (!is.null(.getStateContainerF(jaspResults)[["itemSdObj"]]$object))
-    return(.getStateContainerF(jaspResults)[["itemSdObj"]]$object)
+.frequentistSdItem <- function(jaspResults, dataset, options, model) {
+  if (!is.null(.getStateContainerF(jaspResults)[["sdItemObj"]]$object))
+    return(.getStateContainerF(jaspResults)[["sdItemObj"]]$object)
 
-  out <- model[["itemSd"]]
+  out <- model[["sdItem"]]
   if (is.null(out))
     out <- list()
   # is box even checked?
-  if (options[["itemSd"]]  && is.null(model[["empty"]])) {
+  if (options[["sdItem"]]  && is.null(model[["empty"]])) {
 
     out[["itemDropped"]] <- apply(dataset, 2, sd, na.rm = TRUE)
 
     if (options[["disableSampleSave"]])
       return(out)
 
     stateContainer <- .getStateContainerF(jaspResults)
-    stateContainer[["itemSdObj"]] <- createJaspState(out, dependencies = c("itemSd"))
+    stateContainer[["sdItemObj"]] <- createJaspState(out, dependencies = c("sdItem"))
```

Comment:
```suggestion
    stateContainer[["sdItemObj"]] <- createJaspState(out, dependencies = "sdItem")
```

Reply:
thanks for the tip. I implemented the changes, however, the liner package did not complain about the unneeded concatenation somehow. I removed it anyhow.

Reply:
it's might not be a default lintr, it could be that you need to specify that check manually.